lib/puppet.rb

```diff
@@ -1,6 +1,6 @@
 require 'puppet/version'
 
-if RUBY_VERSION < "1.9.3"
```

Comment:
Is it always safe to reference `Gem` across different supported ruby versions? Not sure if rubygems has to be loaded first?